src/nft/components/collection/Card.tsx

```diff
@@ -536,7 +536,12 @@ const ProfileNftDetails = ({ asset, isSellMode }: ProfileNftDetailsProps) => {
         {asset.susFlag && <Suspicious />}
       </Row>
       <TruncatedTextRow className={subhead} style={{ color: themeVars.colors.textSecondary }}>
-        {!!asset.floorPrice && isSellMode && <span>{`${floorFormatter(asset.floorPrice)} ETH Floor`}</span>}
+        {asset.notForSale && isSellMode && !!asset.floorPrice && (
+          <span>{`${floorFormatter(asset.floorPrice)} ETH Floor`}</span>
+        )}
+      </TruncatedTextRow>
+      <TruncatedTextRow className={subhead} style={{ color: themeVars.colors.textPrimary }}>
+        {!asset.notForSale && <span>{`${floorFormatter(asset.floor_sell_order_price)} ETH`}</span>}
```

Comment:
nit: In one TruncatedTextRow you could do a conditional for asset.notForSale ? {additional checks and floor price} : {floor_sell_order_price} to cut down on repeated code

Reply:
sounds good done, initially i made them separate because of the styling differences but just added a ternary for color.